tests/unit/backend/diagnostics/test_tilt.py

```diff
@@ -220,7 +220,7 @@ def test_calculate_shift():
 
 def test_find_180_deg_pairs():
     # prepare the input list of angles in radians
-    omegas = np.random.random(5) * np.pi
+    omegas = np.sort(np.random.random(5) * np.pi)
```

Comment:
I'm not sure if this particular change is still needed.

Please add a test for having the `atol` specified still.